src/Npgsql.Json.NET/Internal/JsonNetTypeHandlerResolverFactory.cs

```diff
@@ -23,7 +24,11 @@ public class JsonNetTypeHandlerResolverFactory : TypeHandlerResolverFactory
         _jsonClrTypes = jsonClrTypes ?? Array.Empty<Type>();
         _settings = settings ?? new JsonSerializerSettings();
 
-        _byType = new();
+        _byType = new()
+        {
+            { typeof(JObject), "jsonb" },
```

Comment:
I wonder if we should have a config option to determine whether to map to jsonb or json by default... But I don't think it's critical, even if we had one, I think we'd want to default to jsonb, so we can always add that later without it being a breaking change.

Reply:
With an implicit cast, I wouldn't say it's really necessary (we already map JsonDocument to jsonb and no one complained)